source/docs/networking/networking-introduction/windows-firewall-configuration.rst

```diff
@@ -6,51 +6,95 @@ Many of the programming tools used in FRC need network access for various reason
+
+      .. image:: images/windows-firewall-configuration-1.png
+
+      Click :guilabel:`Start` -> :guilabel:`Control Panel` to open the Control Panel. Click the dropdown next to :guilabel:`View by:` and select :guilabel:`Small icons` then click :guilabel:`Windows Defender Firewall`.
+
+      .. image:: images/windows-firewall-configuration-2.png
+
+      In the left pane, click :guilabel`Turn Windows Defender Firewall on or off`, and click yes. Enter your Administrator password if a dialog appears.
+
+      .. image:: images/windows-firewall-configuration-3.png
+
+      For each category, select the radio button to :guilabel:`Turn off Windows Defender Firewall`. Then click OK.
+
+Whitelisting Apps
+-----------------
+
+.. tabs::
+
+   Alternatively, you can add exceptions to the Firewall for any FRC programs you are having issues with.
+
+   .. group-tab:: Windows 10
 
-.. image:: images/windows-firewall-configuration-1.png
+      Click :guilabel:`Start` -> :guilabel:`Settings`
 
-Click ``Start`` -> ``Control Panel`` to open the Control Panel. Click the dropdown next to ``View by:`` and select ``Small icons`` then click ``Windows Defender Firewall``.
+      .. image:: images/windows-firewall/select-settings.png
 
-Turn Windows Firewall on or off
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+      Click :guilabel:`Update & Security`
 
-.. image:: images/windows-firewall-configuration-2.png
+      .. image:: images/windows-firewall/select-update-security.png
 
-In the left pane, click ``Turn Windows Defender Firewall on or off``, and click yes. Enter your Administrator password if a dialog appears.
+      In the left pane, select :guilabel:`Open Windows Security`
 
-Disable the Firewall
-^^^^^^^^^^^^^^^^^^^^
+      .. image:: images/windows-firewall/open-windows-security.png
 
-.. image:: images/windows-firewall-configuration-3.png
+      In the left pane, select :guilabel:`Firewall and network protection`
 
-For each category, select the radio button to ``Turn off Windows Defender Firewall``. Then click OK.
+      .. image:: images/windows-firewall/select-firewall.png
 
-Configure the firewall
-----------------------
+      At the bottom of the window, select :guilabel:`Allow an app through firewall`
 
-Alternatively, you can add exceptions to the Firewall for any FRC programs you are having issues with.
+      .. image:: images/windows-firewall/add-to-firewall.png
 
-Open Control Panel
-^^^^^^^^^^^^^^^^^^
+      For each FRC program you are having an issue with, make sure that it appears in the list and that it has a check in each of the 3 columns. If you need to change a setting, you made need to click the :guilabel:`Change settings` button in the top right before changing the settings. If the program is not in the list at all, click the :guilabel:`Allow another program...` button and browse to the location of the program to add it.
 
-.. image:: images/windows-firewall-configuration-1.png
+      .. image:: images/windows-firewall-configuration-6.png
```

Comment:
Windows 10 screenshot?

Reply:
It's the same look on Windows 10

Reply:
Mine looks slightly different. However, even if the content is the same, the the theme change is jarring